microsoft/nni
# Fix ddp pruner problem

### Description ###

#### Test Options ####
  - [ ] fast test
  - [ ] full test - HPO
  - [ ] full test - NAS
  - [ ] full test - compression

### Checklist ###

### How to test ###
use taylor and activationbased pruners to compress ddp model, then check that masks are the same on different gpus.
